core/entity/level/utils/BuilderAlgorithms.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using WorldWizards.core.controller.level;
+using WorldWizards.core.controller.level.utils;
+using WorldWizards.core.entity.common;
+using WorldWizards.core.entity.coordinate;
+using WorldWizards.core.entity.coordinate.utils;
+using WorldWizards.core.entity.gameObject;
+using WorldWizards.core.entity.gameObject.utils;
+using WorldWizards.core.manager;
+using Object = UnityEngine.Object;
+
+namespace WorldWizards.core.entity.level.utils
+{
+    public static class BuilderAlgorithms
+    {
+        public static void BuildPerimeterWalls(string resourceTag, WWObject wwObject)
+        {
+            var curIndex = wwObject.GetCoordinate().index;
+            var walls = SelectPerimeter(curIndex);
```

Comment:
Please only use var for explicit types.

Reply:
If you feel this, way please change the code formatter. Most people would say type inferencing is good. I do not mind, we can change the formatter.